contrib/IECoreAlembic/src/IECoreAlembic/AlembicScene.cpp

```diff
@@ -700,7 +956,7 @@ IECore::ConstObjectPtr AlembicScene::readAttribute( const Name &name, double tim
 
 void AlembicScene::writeAttribute( const Name &name, const IECore::Object *attribute, double time )
 {
-	throw IECore::NotImplementedException( "AlembicScene::writeAttribute" );
+	IECore::msg( IECore::Msg::Warning, "AlembicScene::writeAttribute", "Not implemented" );
```

Comment:
This next Gaffer release already represents quite a shift, what with C++11 and Qt5, so if possible I'd like to get that rolled out first. I've got Gaffer building/loading with your image work though - was pretty straightforward so far. Got some problems with image colour space and still need to get all the tests passing though...